conf/test_nobc_nodx_vc.config

```diff
@@ -0,0 +1,26 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run nf-core/nanoseq -profile test_nobc_nodx_vc,<docker/singularity>
+ */
+
+params {
+  config_profile_name        = 'Test profile'
+  config_profile_description = 'Minimal test dataset to check variant calling functions'
+
+  // Limit resources so that this can run on Travis
+  max_cpus            = 2
+  max_memory          = 6.GB
+  max_time            = 12.h
+
+  // Input data to skip basecalling and demultiplexing, and variant call
+  input               = 'https://raw.githubusercontent.com//christopher-hakkaart/testdata/main/samplesheet_nobc_vc.csv'
```

Comment:
It will be great if the test dataset (including the `samplesheet.csv`) can be moved to [`nf-core/test-dataset`](https://github.com/nf-core/test-datasets/tree/nanoseq)

Reply:
Pull request has been submitted to move into the nf-core/test-dataset nanoseq branch. Once it's accepted I will edit the config to correct for this.